docs/how-to-guides/advanced-user-guides/cis-scan-guides/install-rancher-cis-benchmark.md

```diff
@@ -15,3 +19,9 @@ title: Install Rancher CIS Benchmark
 CIS Benchmark 4.0.0 and above have PSPs disabled by default. To install CIS Benchmark on a hardened cluster, set `global.psp.enabled` to `true` in the values before installing the chart.
 
 :::
+
+</TabItem>
+<TabItem value = "Kubernetes v1.25+">
+If you have a cluster running Kubernetes v1.25 or later, you can set the Rancher CIS benchmark by [assigning](../../new-user-guides/authentication-permissions-and-global-configuration/psa-config-templates.md) the `rancher-restricted` [PSA](../../new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md) configuration template to the cluster and selecting a CIS profile.  
```

Comment:
This page is about installing the CIS Benchmark Helm Chart (and the note is about modifying the chart), which is a different topic than the content being added to this tab. If we're addressing #755 using tabs, this tab should have the same instructions minus the note.

Also, we should have the more recent tab come first.

Reply:
But if the instructions only apply to PSP-hardened clusters, how could they apply to clusters running Kubernetes v1.25 and later? PSPs were removed in v1.25. @Priyashetty17

Reply:
> But if the instructions only apply to PSP-hardened clusters, how could they apply to clusters running Kubernetes v1.25 and later? PSPs were removed in v1.25.
> 
> From #755:
> 
> > it is not relevant to PSA hardened clusters (K8s version >= 1.25)
> 
> @Priyashetty17

Hence the `minus the note` bit in my comment above. #755 is referring to the note only, not the entirety of the instructions. The note calls out setting a Helm value to turn on PSPs for the chart and the PSP portion is the only thing not applicable to K8s >= 1.25 on this page. The rest of the step-by-step instructions to install the chart aren't affected by whether its PSP or PSA. The new tab for v1.25+ removes how to install the chart, which is the purpose of the page.